src/Wfirma/Contractor/Factory/ContractorFactory.php

```diff
@@ -22,20 +23,22 @@ final class ContractorFactory
     public function getContractor(array $data): Contractor
     {
         if (empty($data['nip'])) {
-            return new Contractor\Person(
+            return new Person(
                 new Contractor\ContractorIdentifier((string) $data['id']),
                 new ContractorName((string) $data['name']),
                 new Contractor\ContractorEmail((string) $data['email']),
                 $this->getContractorAddress($data)
             );
         }
 
+        $id = str_replace((string) $data['country'], '', (string) $data['nip']);
```

Comment:
This variable name can be misunderstood because of the same identifier usage below for the completely different context  - `$data['id']`. You can completely omit it - there is only one usage of it. However, if you want to keep this variable for some reason, I suggest changing the name to the `$taxIdentifier`.